PWGDQ/TableProducer/tableMaker_withAssoc.cxx

```diff
@@ -478,6 +490,29 @@ struct TableMaker {
       VarManager::fgValues[VarManager::kTimestamp] = bc.timestamp();
       VarManager::fgValues[VarManager::kRunIndex] = VarManager::GetRunIndex(bc.runNumber());
       VarManager::FillEvent<TEventFillMap>(collision); // extract event information and place it in the fValues array
+      if constexpr ((TEventFillMap & VarManager::ObjTypes::Zdc) > 0) {
```

Comment:
Open to cosmetic changes and suggestions regarding this block. Maybe there is somewhere better to put this?

Reply:
I think its fine like this